### PR TITLE
PIM-3583 : Fix the bad parsed filter value with spaces

### PR DESCRIPTION
**Pull request by grena**
| Q                    | A
| -------------------- | ---
| Bug fix?             | Y
| New feature?         | N
| BC breaks?           | N
| CI currently passes? | N
| Tests pass?          | Y
| Scenarios pass?      | Build 1869 running
| Checkstyle issues?*  | N
| PMD issues?**        | N
| Changelog updated?   | Y
| Fixed tickets        | PIM-3583
| DB schema updated?   | N
| Migration script?    | -
| Doc PR               | -

### CODE REVIEW

File: src/Pim/Bundle/UIBundle/Resources/public/js/app.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+            /**
+             * Decode URL encoded component
+             *
+             * @param {String} string
+             * @return {String}
+             * @protected
+             */
+            _decodeComponent: function(string) {
+                var result = string.replace(/\+/g, '%20');
```
Comment (grena) [inline src/Pim/Bundle/UIBundle/Resources/public/js/app.js]: The bug fix is here, original contained : 
```js
var result = string.replace('+', '%20');
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -194,6 +194,7 @@
 - PIM-3548: Do not use the absolute file path of a media
 - PIM-3730: Fix variant group link on product edit page
 - PIM-3632: Correctly show scopable attribute icons on scope change
+- PIM-3583 : Fix the bad parsed filter value with spaces
```
Comment (filipsalpe) [inline CHANGELOG.md]: No space before a colon, ever :)

Comment (grena) [inline CHANGELOG.md]: Niah, you're right, freaking french language reflex :family:

**Comment by filipsalpe** [comment]
GTM

**Pull request merged**